src/SFA.DAS.Assessor.Functions/Domain/Print/Services/NotificationService.cs

```diff
@@ -43,6 +51,7 @@ public async Task Send(int batchNumber, List<Certificate> certificates, string c
 
         private Dictionary<string, string> CreatePersonalisationTokens(List<Certificate> certificates, string certificatesFileName)
         {
+            // TODO: The template which is sent need to be re-worked including the parameters below
```

Comment:
Why does the template need to be re-worked when none of the tokens have changed?

Reply:
The content mentions SFTP in the text